### PR TITLE
PV1-86-subscription-modal-time-format

### PR DESCRIPTION
**Pull request by Bogdancrunch**

### CODE REVIEW

File: src/elements/SubscribeModal/SubscribeModal.vue [hunk 1]
```diff
@@ -278,17 +278,17 @@
             },
             onSaveClick () {
                 this.disabled = true;
-                
+                let time = this.getSendTime();
```
Comment (dylanspurgin) [inline src/elements/SubscribeModal/SubscribeModal.vue:281]: Seems unnecessary to have this assigned to a variable here. Lets just assign directly to `send_at` below

File: src/elements/SubscribeModal/SubscribeModal.vue [hunk 2]
```diff
@@ -278,17 +278,17 @@
...
                 let subscription = {
                     email: this.email,
                     view_name: this.formState.dashboard.id != 0 ? this.getViewName(this.formState.dashboard.url) : this.getViewName(this.currentDashboard.url),
                     schedule: this.formState.frequency, // daily, weekly, monthly
                     day_of: this.formState.frequency === 'DAILY' ? 0 : this.formState.frequency === 'WEEKLY' ? this.formState.dayOfWeek : this.formState.dayOfMonth,
-                    send_at: `${this.formState.hour}:${this.formState.minute} ${this.formState.ampm}`, // "8:00 am"
+                    send_at: time, // "8:00 - 24-h format"
```
Comment (dylanspurgin) [inline src/elements/SubscribeModal/SubscribeModal.vue:287]: Update comment to show 24 hour example: // "16:00"

File: src/elements/SubscribeModal/SubscribeModal.vue [hunk 3]
```diff
@@ -278,17 +278,17 @@
...
                     tz: Intl.DateTimeFormat().resolvedOptions().timeZone,
                     full: this.formState.dashboard.id === 0
                 };
-
+                console.log(subscription);
```
Comment (dylanspurgin) [inline src/elements/SubscribeModal/SubscribeModal.vue:291]: Remove log

Comment (Bogdancrunch) [reply src/elements/SubscribeModal/SubscribeModal.vue:291]: It was removed in the other PR.

**Pull request merged**